docs/diagrams/MatchAutoActivityDiagram.puml

```diff
@@ -0,0 +1,19 @@
+@startuml
+skinparam defaultTextAlignment center
+skinparam DefaultFontSize 13
+
+start
+:User executes `match auto`;
+:Generate all possible matches;
+:Sort candidate matches
+by match score;
```

Comment:
Not sure if more details need to be mentioned here. Technically the price gap factors into the sorting alongside the match score.

Reply:
I think that's a detail. It is true that they are sorted by match score, the guide points to the classes where the full details can be found.